src/DayColumn.js

```diff
@@ -84,6 +84,15 @@ class DayColumn extends React.Component {
     this.slotMetrics = this.slotMetrics.update(nextProps)
   }
 
+  componentDidUpdate(prevProps) {
+    if (prevProps.date.getTime() !== this.props.date.getTime()) {
```

Comment:
The resolution of this comparison should be minutes: `!dates.eq(prevDate, nextDate, 'minutes')

Reply:
I believe that method comes from the package `date-arithmetic` which `utils\dates.js` extends.